test/string.js

```diff
@@ -2983,7 +2983,74 @@ describe('string', () => {
                 ['b4b2fb69c6244e5eb0698e0c6ec66618', true],
                 ['{283B67B2-430F-4E6F-97E6-19041992-C1B0}', false, null, '"value" must be a valid GUID'],
                 ['{D1A5279D-B27D-4CD4-A05E-EFDD53D08E8D', false, null, '"value" must be a valid GUID'],
-                ['D1A5279D-B27D-4CD4-A05E-EFDD53D08E8D}', false, null, '"value" must be a valid GUID']
+                ['D1A5279D-B27D-4CD4-A05E-EFDD53D08E8D}', false, null, '"value" must be a valid GUID'],
```

Comment:
@gelliott181 There is already a test for that: line 2978.

@DavidTPate I see what you are saying and I'll add that in, thanks.